### DISCUSSION TITLE
Seit Update auf 3.67.10.20230114 läuft Duty Cycle voll!

### DISCUSSION THREAD

**Discussion by smartmatic** [OpenCCU/OpenCCU#2180] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

ich nutze Raspberrymatic jetzt seit einem Jahr ohne Probleme. Daher tue ich mir schwer bei der aktuellen Fehlersuche, die ich noch nicht hatte. Seit dem Update gestern 3.67.10.20230114 läuft der Duty Cyle voll!

Ich habe 50 Geräte in Verwendung. Die einzige Änderunge nach dem Update war, dass ich bisher verwendete direkte Verknüpfungen über Gruppen umsetzen wollte. Die direkten Verbindungen hatte ich gelöscht und dann die Gruppen angelegt.

Da der Duty Cyle voll läuft dachte ich es liegt an den neue Gruppen. In der Zwischenzeit habe ich die Gruppen wieder gelöscht und die Geräte über Raspberymatic in den Werkszustand versetzt um sicher zu gehen, dass diese Änderung nicht das Problem verursacht.

Leider ohne Verbesserung beim Duty Cyle. 

Wie kann ich schnell ermitteln, welche Geräte den Duty Cyle nach oben treiben?

**Comment by MichaelN0815** [comment, marked as answer]
Das wird bei dir auch nicht anders sein als bei allen anderen. Daher einfach die bekannten Tipps abarbeiten

https://homematic-forum.de/forum/viewtopic.php?f=26&t=53387#p532517

**Comment by smartmatic** [comment]
Die Lösung war die Geräte zu resetten. Scheinbar hat das löschen der direkten Verbindungen die Verbindungen auf den Geräten nicht gelöscht. Davon bin ich aber ausgegangen, weil sie zur Gruppenbildung zur Verfügung standen. Jedenfalls konnte die Konfiguration der Gruppe nie abgeschlossen werden. 
Also Reset > Neu anlernen > Gruppe bilden > Konfigurationsdaten übertragen.
